### DISCUSSION TITLE
How to specifically define the color levels used by `colorScheme`

### DISCUSSION THREAD

**Discussion by neil-morgan** [chakra-ui/chakra-ui#2290] [Q&A]
Let's say I have a button and I want to use a custom color defined by a brand.
Chakra needs the color split into an object with stops ranging from 50-900.  (50, 100 ,200 etc...)
It seems that the default for light mode is set to 500 (so that's where to set the base brand color)

```
<Button
    bg={
        colorMode === 'light'
            ? 'primary.500'
            : 'tertiary.500'
    }
>
    Button
</Button>
```

**Comment by chasinhues** [comment]
You could implement a variant of button that changes the colour scheme used based on the colour mode.

```jsx
const theme = extendTheme({
  components: {
    Button: {
      variants: ({ colorMode }) => ({
        custom: {
          bg: colorMode === "light" ? "primary.500" : "tertiary.500",
        }
      })
    }
  }
});
```

Then to use this button would simply be a matter of:
```jsx
<Button variant="custom">My Custom Button</Button>
```
